### DISCUSSION TITLE
When downloading Osu Laser version 2024.1115.3 for Android, version 2024.1115.1 is downloaded. Please fix it.

### DISCUSSION THREAD

**Discussion by afa555** [ppy/osu#30719] [Q&A] [answered]
When downloading Osu Laser version 2024.1115.3 for Android, version 2024.1115.1 is downloaded. Please fix it.

**Comment by felipemarins** [comment, marked as answer]
2024.1115.3 is just a fix for the Linux build, so you shouldn't worry about it. Just ignore the update notifications for now, I guess.
